lib/jekyll-plugin-platoniq-journal.rb

```diff
@@ -2,6 +2,8 @@
 
 require "jekyll"
 require "kramdown"
+require "support/localized_block"
+require "support/localized_tag"
```

Comment:
As both files are _nearly_ identical, we could create a `Base` module  that contain some methods common to both `tags` and `blocks`:

```suggestion
require "jekyll-plugin-platoniq-journal/base"
```